apps/workflowengine/src/components/Checks/file.js

```diff
@@ -100,6 +100,17 @@ const FileChecks = [
 		],
 		component: FileSystemTag,
 	},
+
+	{
+		class: 'OCA\\WorkflowEngine\\Check\\MfaVerified',
+		name: t('workflowengine', 'Mfa Verified'),
```

Comment:
I wonder whether we should write 'Mfa' or 'MFA'.

Reply:
I suggest MFA

Reply:
I'll check it too on other sites.

Reply:
https://en.wikipedia.org/wiki/Multi-factor_authentication

I think MFA too.